packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.17.1
```

Comment:
After the pubspec is updated this will also need to be updated.

Reply:
I think this was addressed with https://github.com/flutter/packages/pull/9674/commits/8b9878efa1e4d8ce996c90c7bcdf300bf6b8d1f8 and is already at 2.18.0.

Reply:
I am sorry I see the changes now I must have had an older versions cached in my browser.